src/resources/authoring/gv-1.json

```diff
@@ -2195,7 +2195,96 @@
             "text": "Start of mission 6.3"
           }]
         },
-        "name": "Mission 6.3"
+        "name": "Mission 6.4"
```

Comment:
This should be "Mission 6.3"

Reply:
Good catch. Over-aggressive copy-replace. Fixed.